pkg/util/kubernetes/kubeadm/app/constants/constants_unix.go

```diff
@@ -1,3 +1,4 @@
+//go:build !windows
```

Comment:
Could we remove this tag?
It seems to be in use since go1.17, while openyurt is still using go1.16. We can add it when we bump go version to 1.17 or above.

Reply:
ok